receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_test.go

```diff
@@ -38,7 +38,13 @@ const (
 	bootTime     = 100
 )
 
+// Skips test without applying unused rule
+var skip = func(t *testing.T, why string) {
```

Comment:
Why this func is needed, just `t.Skip` -> `skip`? I would avoid adding it

Reply:
The comment explains it, but basically because skipping the function causes multiple lint issues due to structs/funcs becoming unused. I think this is better and less effort than chasing down and modifying the unused parts.

Reply:
Ok sounds good

Reply:
Thanks for explaining